### PR TITLE
Added login page, vite and made small changes

### PR DESCRIPTION
**Pull request by msaivasanth**
Added login page and vite sever to project 
Made small changes to some other files

### CODE REVIEW

File: package-lock.json
Comment (syedzeeshan1) [inline package-lock.json:17]: create a .gitignore file and add package-lock.json to it.

files like this or node_modules should never be pushed to your repository.

File: src/Components/Login.tsx
Comment (syedzeeshan1) [inline src/Components/Login.tsx]: try to separate out API calls in a different service file. keep the code organized

File: src/Components/Login.tsx
Comment (syedzeeshan1) [inline src/Components/Login.tsx]: nit: remove unnecessary comments

**Review by syedzeeshan1 — CHANGES_REQUESTED** [review]
Overall looks good, minor changes are requested.

Also, try to see if you can deploy this app for free through some website like this - https://www.netlify.com/with/react/

This way I will also be able to look at UI and test e2e

**Comment by syedzeeshan1** [comment]
@msaivasanth avoid merging without approval. 
LGTM, but why is the Vercel inspect check failing?

**Comment by msaivasanth** [comment]
> @msaivasanth avoid merging without approval. LGTM, but why is the Vercel inspect check failing?

I apologize for merging without approval. Vercel inspect check failing because, first I deployed the project without merging the login branch at that time the project does not contain vite and at the deployment framework was react, after merging with login branch now it contains vite but the framework was still react for project, so the first check failed because it directly tried to deploy with merged commit which contains vite and framework was react in vercel. So I deleted the previous deployment and the I deployed the repository which now contains vite and specified vite as framework so second check was successful.

**Pull request merged**
